internal/subshell/sscommon/rcfile.go

```diff
@@ -274,7 +274,7 @@ func SetupProjectRcFile(prj *project.Project, templateName, ext string, env map[
 
 	isConsole := ext == ".bat" // yeah this is a dirty cheat, should find something more deterministic
 
-	activatedMessage := txtstyle.NewTitle(locale.Tl("youre_activated", "You're Activated!"))
+	activatedMessage := txtstyle.NewTitle(locale.Tl("language_installed", "{{.V0}} has been sucessfully activated", prj.Name()))
```

Comment:
This needs to account for the project been headless. Maybe for that we can just say "Your virtual environment has been successfully activated".

Reply:
I think that makes sense. I don't think there is a way for a headless project to be activated as default so `virtual environment` in the messaging should work. Do we want to get product sign-off on this?